frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts

```diff
@@ -44,11 +45,11 @@ export class VirtualMachine extends KubevirtDetailView {
     await this.navigateToTab(TABS.OVERVIEW);
 
     let confirmDialog = true;
-    if (['Clone'].includes(action)) {
+    if ([VMACTIONS.CLONE].includes(action)) {
```

Comment:
`VmActions` or `VM_ACTIONS` is more readable

Reply:
Done